### PR TITLE
refactor(msgs): remove one layer of indirection

### PR DESCRIPTION
**Pull request by oetyng**
- `ValidateMsg` can be replaced with `HandleMsg`.

### CODE REVIEW

File: sn_node/src/node/flow_ctrl/tests/mod.rs [hunk 1]
```diff
@@ -1498,12 +1498,12 @@ async fn spentbook_spend_with_updated_network_knowledge_should_update_the_node()
     )
     .await?;
 
-    // The commands returned here should include the new command to update the network
-    // knowledge and also the other two commands to replicate the spent proof shares and
-    // the ack command, but we've already validated the other two as part of another test.
-    assert_eq!(cmds.len(), 3);
-    let update_cmd = cmds[0].clone();
-    assert_matches!(update_cmd, Cmd::UpdateNetworkAndHandleValidClientMsg { .. });
+    // // The commands returned here should include the new command to update the network
```
Comment (joshuef) [inline sn_node/src/node/flow_ctrl/tests/mod.rs:1501]: Think we're missing to update this test, no?

Comment (oetyng) [reply sn_node/src/node/flow_ctrl/tests/mod.rs:1501]: Indeed. But it is already ignored. Not sure it's worth it since it will be reimplemented.

Comment (joshuef) [reply sn_node/src/node/flow_ctrl/tests/mod.rs:1501]: ah the containing thing here is ignored? Aye right cool.

should we remove the commented stuff or it's there until re-impl time as a reminder?

**Review by joshuef — APPROVED** [review]

**Pull request merged**